samcli/commands/deploy/guided_config.py

```diff
@@ -26,13 +27,17 @@ def get_config_ctx(self, config_file=None):
         return ctx, samconfig
 
     def read_config_showcase(self, config_file=None):
-        _, samconfig = self.get_config_ctx(config_file)
-
-        status = "Found" if samconfig.exists() else "Not found"
         msg = (
             "Syntax invalid in samconfig.toml; save values "
             "through sam deploy --guided to overwrite file with a valid set of values."
         )
+        try:
+            _, samconfig = self.get_config_ctx(config_file)
+        except SamConfigFileReadException:
+            raise GuidedDeployFailedError(msg)
+
+        status = "Found" if samconfig.exists() else "Not found"
```

Comment:
I think letting the `status` be an enum might be a bit of overkill, since it's only used in this one method, and only really exists to be appended to the following click message. We could make it an enum, but I don't think it's necessary in this case.

Reply:
Having strings be status seems counter-intuitive to me, won't block on it. but we should leave a todo at the very least.